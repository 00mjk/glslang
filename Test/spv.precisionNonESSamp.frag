#version 450

precision lowp sampler2D;
precision lowp int;
precision lowp float;

uniform lowp sampler2D s;

layout(location = 0) in lowp vec2 v;

layout(location = 0) out lowp vec4 color;

void main()
{
    color = texture(s, v);
}
