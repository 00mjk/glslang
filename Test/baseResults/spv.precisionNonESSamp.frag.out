spv.precisionNonESSamp.frag
// Module Version 10000
// Generated by (magic number): 80002
// Id's are bound by 20

                              Capability Shader
               1:             ExtInstImport  "GLSL.std.450"
                              MemoryModel Logical GLSL450
                              EntryPoint Fragment 4  "main" 9 17
                              ExecutionMode 4 OriginUpperLeft
                              Source GLSL 450
                              Name 4  "main"
                              Name 9  "color"
                              Name 13  "s"
                              Name 17  "v"
                              Decorate 9(color) RelaxedPrecision
                              Decorate 9(color) Location 0
                              Decorate 13(s) RelaxedPrecision
                              Decorate 13(s) DescriptorSet 0
                              Decorate 14 RelaxedPrecision
                              Decorate 17(v) RelaxedPrecision
                              Decorate 17(v) Location 0
                              Decorate 18 RelaxedPrecision
                              Decorate 19 RelaxedPrecision
               2:             TypeVoid
               3:             TypeFunction 2
               6:             TypeFloat 32
               7:             TypeVector 6(float) 4
               8:             TypePointer Output 7(fvec4)
        9(color):      8(ptr) Variable Output
              10:             TypeImage 6(float) 2D sampled format:Unknown
              11:             TypeSampledImage 10
              12:             TypePointer UniformConstant 11
           13(s):     12(ptr) Variable UniformConstant
              15:             TypeVector 6(float) 2
              16:             TypePointer Input 15(fvec2)
           17(v):     16(ptr) Variable Input
         4(main):           2 Function None 3
               5:             Label
              14:          11 Load 13(s)
              18:   15(fvec2) Load 17(v)
              19:    7(fvec4) ImageSampleImplicitLod 14 18
                              Store 9(color) 19
                              Return
                              FunctionEnd
